### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by ssbarnea** [pycontribs/selinux#62] [Announcements]
## Minor Changes

- Require python>=3.9 (#61) @ssbarnea

## Bugfixes

- Require python>=3.9 (#61) @ssbarnea
- Replace Zuul with GHA (#60) @ssbarnea
- Change system_python variable to consider minor python version (#56) @dmasteller4
- Upgrade to GitHub-native Dependabot (#47) -preview
- Consolidate dependencies (#54) @dvzrv
- Fix typo in README.md (#46) @roaldnefs
- Code refresh (#45) @ssbarnea
- Drop Travis (#44) @ssbarnea
- Enable Zuul CI integration on selinux (#37) @ssbarnea
- Bump pre-commit (#39) @ssbarnea

<hr /><em>This discussion was created from the release <a href='https://github.com/pycontribs/selinux/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by sophos-rickc** [comment]
Are we the only organization whose Ansible installations on CentOS Stream 8 were broken by this update? We have Python 3.9 installed on a custom path, not `/usr/bin/python3.9`. As soon as Ansible starts, it aborts with an exception. This is, essentially, this same issue as #35 and #64 ...

WE NEED THIS FIXED ASAP!

Surely, we can't be the only organization experiencing this..

**Reply by ssbarnea** [reply]
Dear @sophos-rick, as I do not see you as having a contributor or sponsor near your username, I may try to suggest using https://github.com/sponsors/ssbarnea?frequency=one-time&sponsor=ssbarnea as a way to motivate others to fix bugs in open-source projects that affect you.

**Reply by dmasteller4** [reply]
Created a PR #65 for this issue. Should revert to previous functionality in cases where custom paths are used
